### DISCUSSION TITLE
New bluetooth devices and zzh cc2652R

### DISCUSSION THREAD

**Discussion by kitopopo** [Koenkk/zigbee2mqtt#11705] [Devices]
Dear @Koenkk ,

In the market are appearing  some bluethooth devices, for example xiaomi smoke detectir JTYJ-GD-03MI/BB (bluetooth 5.0 version ). I have a zzh cc2652r with bluetooth. In the pressent is possible add bluetooth devices? Are you working for can use bluetooth devices with zzh? How i can search more info? Xiaomi have a gateway with zigbee and bluetooth. 
Yesterday i can see a new xiaomi bluetooth device. Door and window sensor 2. MCCGQ02HL model. Whats happens with the bluetooth devices? Thanks in advanced

**Comment by Koenkk** [comment]
I'm only focussing on Zigbee, therefore bluetooth is out-of-scope for zigbee2mqtt.

**Reply by SeveranExp** [reply]
A little bit of an off topic but, have you come about zigbee devices running on esp32 that use ble for the initial configuration and pairing?
I’ve been wondering how we’d get those to pair with zigbee2mqtt..

**Reply by Koenkk** [reply]
No I haven't seen/heard about those devices.
